src/main/java/ca/corefacility/bioinformatics/irida/ria/config/thymeleaf/webpacker/util/WebpackAssetsManifestDeserializer.java

```diff
@@ -0,0 +1,50 @@
+package ca.corefacility.bioinformatics.irida.ria.config.thymeleaf.webpacker.util;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+/**
+ * Deserializer for the webpack assets-manifest file
+ */
+public class WebpackAssetsManifestDeserializer extends StdDeserializer<WebpackAssetsManifest> {
+
+	public WebpackAssetsManifestDeserializer() {
+		super(WebpackAssetsManifest.class);
+	}
+
+	@Override
+	public WebpackAssetsManifest deserialize(JsonParser jsonParser, DeserializationContext deserializationContext)
+			throws IOException {
+		ObjectMapper mapper = new ObjectMapper();
+		SimpleModule module = new SimpleModule();
+		module.addDeserializer(WebpackEntry.class, new WebpackEntryDeserializer());
+		mapper.registerModule(module);
+
+		TreeNode treeNode = jsonParser.readValueAsTree();
+
+		Map<String, WebpackEntry> entries = new HashMap<>();
+		final TreeNode entrypoints = treeNode.get("entrypoints");
+		entrypoints
+				.fieldNames()
+				.forEachRemaining(name -> {
+					try {
+						WebpackEntry entry = mapper.readValue(entrypoints.get(name).toString(), WebpackEntry.class);
+						entries.put(name, entry);
+					} catch (JsonProcessingException e) {
+						e.printStackTrace();
+					}
```

Comment:
Please don't just catch and swallow this exception.  It should either be handled, wrapped & rethrown, or just allowed to propagate up if it's a real error.

Reply:
Logged as error the exception with SL4J logging.  Cannot seem to do much else inside the deserializer, let me know if you have other ideas.

Reply:
Well I guess my point there is if one of the entries fails JSON parsing what should happen?  Right now it would just log the error and skip that entry, but still continue with the rest normally and start the server.  I feel like if something in this build fails to parse, it's a big enough problem that the whole process should fail.  That `JsonProcessingException` should likely be allowed to propagate up instead of being swallowed and logged here.  The method already throws `IOException` so whatever's above it will handle it.

Reply:
Updated in ef49dd1